Bottom: dc6cfa9bcb31aa0a41af666faf5c771c287d6647
Top:    8c5f94660d224e46de9ed5f55b43153f0ccba9a2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:17 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
